File: app/src/main/java/com/kml/searchEngine/internalRoomDatabase/Game.java
package com.kml.searchEngine.internalRoomDatabase;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "gameTable")
public class Game
{
    @PrimaryKey(autoGenerate = true)
    private int id;
    private String name;
    private String description;
    private String requirements;
    private String numberOfKids;
    private String kidsAge;
    private String place;
    private String typeOfGame;
    private String category;
    public Game(String name, String description, String requirements, String numberOfKids, String kidsAge, String place, String typeOfGame, String category)
    {
        this.name = name;
        this.description = description;
        this.requirements = requirements;
        this.numberOfKids = numberOfKids;
        this.kidsAge = kidsAge;
        this.place = place;
        this.typeOfGame = typeOfGame;
        this.category = category;
    }
    public void setId(int id)
    {
        this.id = id;
    }
    public int getId()
    {
        return id;
    }

    public String getName()
    {
        return name;
    }
    public String getDescription()
    {
        return description;
    }
    public String getNumberOfKids()
    {
        return numberOfKids;
    }

    public String getRequirements()
    {
        return requirements;
    }
    public String getKidsAge()
    {
        return kidsAge;
    }
    public String getTypeOfGame()
    {
        return typeOfGame;
    }
    public String getPlace()
    {
        return place;
    }
    public String getCategory() {
        return category;
    }
}

File: app/src/main/java/com/kml/data/models/Game.kt
package com.kml.data.models

import android.os.Parcelable
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.android.parcel.IgnoredOnParcel
import kotlinx.android.parcel.Parcelize

@Parcelize
@Entity(tableName = "gameTable")
class Game(var name: String,
           var description: String,
           var requirements: String,
           var numberOfKids: String,
           var kidsAge: String,
           var place: String,
           var typeOfGame: String,
           var category: String) : Parcelable {

    @IgnoredOnParcel
    @PrimaryKey(autoGenerate = true)
    var id = 0
}